.github/workflows/tag-uploads.yml

```diff
@@ -54,7 +54,26 @@ jobs:
     runs-on: [ubuntu-latest]
     steps:
     - uses: actions/checkout@v3
-    - uses: pypa/gh-action-pypi-publish@release/v1
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.x'
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
```

Comment:
I tried this command on my laptop, I think you need to `cd` to `py` directory first.

Reply:
good catch, thanks